Source/Shaders/Vector3DTileClampedPolylinesVS.glsl

```diff
@@ -0,0 +1,86 @@
+attribute vec3 startEllipsoidNormal;
+attribute vec3 endEllipsoidNormal;
+attribute vec4 startPositionAndHeight;
+attribute vec4 endPositionAndHeight;
+attribute vec4 startFaceNormalAndVertexCorner;
+attribute vec4 endFaceNormalAndHalfWidth;
+attribute float a_batchId;
+
+uniform mat4 u_modifiedModelView;
+uniform vec2 u_minimumMaximumVectorHeights;
+
+varying vec4 v_startPlaneEC;
+varying vec4 v_endPlaneEC;
+varying vec4 v_rightPlaneEC;
+varying float v_halfWidth;
+varying vec3 v_volumeUpEC;
+
+void main()
+{
+    vec3 scratchNormal;
```

Comment:
I think this originates from some paranoia that GLSL doesn't optimize how registers get used, but in practice it's probably fine to keep creating new `vec3` vars instead. I think that would be more readable.

Reply:
@lilleyse any thoughts on this?

Reply:
I agree with @likangning93, should be fine to create a new `vec3` every time instead instead of a scratch.